icons/nodejs/nodejs-plain-wordmark.svg

```diff
@@ -1 +1 @@
 1 .752 0l3.846 2.213a.762.762 0 0 1 .378.64v3.494c0 .263.144.525.375.641a.704.704 0 0 0 .754 0l7.291-4.28a1.46 1.46 0 0 0 .725-1.282V51.22c0-.524-.29-1.017-.725-1.28l-12.183-7.103a1.499 1.499 0 0 0-.739-.2zm-67.101.052a1.359 1.359 0 0 0-.777.202l-12.155 7.046.014.024-.014.008a1.456 1.456 0 0 0-.724 1.281v14.18c0 .32.108.629.29.88l-.029.051c.116.146.26.264.405.352l12.242 7.074c.283.166.603.23.916.188l.01.015c.173-.029.376-.086.55-.174l12.211-7.105c.348-.204.609-.553.725-.932l-.053-.092c.014-.084.024-.17.024-.257v-14.18c0-.372-.148-.729-.387-.998l.012-.022a.905.905 0 0 0-.32-.261l-6.12-3.54-6.092-3.538a1.484 1.484 0 0 0-.46-.166l-.004-.008a1.885 1.885 0 0 0-.264-.028zm33.504 10.582c.065 0 .13.015.19.045l4.193 2.448c.116.058.175.201.175.318v4.893c0 .146-.06.264-.175.322l-4.196 2.445a.431.431 0 0 1-.377 0L76.373 61.3c-.116-.058-.176-.205-.176-.322v-4.893c0-.146.061-.261.176-.32l4.195-2.446a.425.425 0 0 1 .19-.045zm33.555 1.98a.26.26 0 0 0-.145.046l-2.346 1.369a.3.3 0 0 0-.142.26v2.74c0 .116.055.204.142.262l2.346 1.369a.262.262 0 0 0 .289 0l2.344-1.37a.308.308 0 0 0 .144-.261V56.93a.303.303 0 0 0-.144-.26l-2.344-1.371a.26.26 0 0 0-.144-.043v-.002zM63.22 71.64c-.427 0-.853.102-1.215.306l-11.549 6.729a2.457 2.457 0 0 0-1.215 2.123V94.22c0 .874.463 1.689 1.215 2.127l3.04 1.746c1.475.728 1.996.726 2.661.726 2.17 0 3.416-1.34 3.416-3.64V81.936a.356.356 0 0 0-.347-.352h-1.475a.356.356 0 0 0-.35.352v13.248c0 1.019-1.068 2.039-2.775 1.166l-3.156-1.834c-.116-.058-.174-.207-.174-.323V80.768c0-.116.058-.262.174-.32l11.545-6.696c.087-.058.233-.058.35 0l11.546 6.695c.115.06.174.175.174.32v13.424c0 .145-.059.265-.174.323l-11.547 6.726c-.087.058-.233.058-.35 0l-2.95-1.777c-.088-.058-.203-.087-.29-.03-.81.467-.952.528-1.734.79-.174.058-.463.172.115.492l3.85 2.303c.376.203.78.318 1.215.318.434 0 .866-.115 1.213-.26l11.548-6.726a2.463 2.463 0 0 0 1.215-2.13V80.798c0-.874-.463-1.686-1.215-2.123l-11.548-6.729a2.488 2.488 0 0 0-1.213-.306zm18.029 6.129a2.236 2.236 0 0 0-2.227 2.244 2.236 2.236 0 0 0 2.227 2.242c1.217 0 2.229-1.02 2.229-2.242a2.254 2.254 0 0 0-2.229-2.242v-.002zm-.03.378a1.86 1.86 0 0 1 1.884 1.866c0 1.02-.841 1.892-1.883 1.892-1.012 0-1.852-.844-1.852-1.892s.869-1.866 1.852-1.866zm-.81.612v2.562h.494v-1.015h.436c.174 0 .23.058.26.203 0 .03.086.669.086.785h.52c-.059-.116-.087-.466-.116-.67-.028-.32-.056-.553-.404-.582.174-.059.462-.145.462-.611 0-.67-.58-.67-.869-.67l-.869-.002zm.436.408h.404c.146 0 .377 0 .377.35 0 .116-.057.35-.377.35h-.404v-.7zm-14.469 2.01c-3.3 0-5.27 1.399-5.27 3.758 0 2.534 1.968 3.23 5.123 3.55 3.79.38 4.08.933 4.08 1.69 0 1.31-1.043 1.863-3.474 1.863-3.068 0-3.733-.758-3.965-2.3 0-.176-.142-.29-.316-.29H61.05a.35.35 0 0 0-.348.35c0 1.98 1.042 4.308 6.108 4.308 3.645 0 5.757-1.457 5.757-4.02 0-2.504-1.68-3.174-5.238-3.64-3.59-.466-3.965-.728-3.965-1.572 0-.699.319-1.629 2.98-1.629 2.374 0 3.27.524 3.618 2.125a.34.34 0 0 0 .318.262h1.533c.088 0 .176-.057.235-.115a.476.476 0 0 0 .086-.264c-.231-2.795-2.053-4.076-5.758-4.076z"/></svg>
```

Comment:
This too should be changed so that the pattern thingy looks more like what I suggested on nodejs-plain